apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -212,8 +210,6 @@ cluster_query(Tab, QString, QSchema, MsFun, FmtFun) ->
     end.
 
 %% @private
-do_cluster_query([], _QueryState, ResultAcc) ->
-    ResultAcc;
```

Comment:
I think we could keep this clause.  When `mria` stops (e.g.: when it restarts for a node to join a cluster), then `mria_mnesia:running_nodes/0` will return an empty list.

Reply:
Brought the clause back.